app/views/partials/resultat.html

```diff
@@ -84,6 +84,14 @@
               Plus d'informations
             </a>
           </div>
+          <div ng-if="droit.id == 'paris_logement_familles'"
+            class="col-sm-3 col-sm-offset-6">
+            <a target="_self"
+              class="btn btn-sm btn-primary btn-block"
+              href="/documents/Formulaire_demande_PLF.pdf" download>
```

Comment:
@MattiSG,
I know `download` doesn't work for all browsers, but do you have any idea to let the bénéficiaire potentiel access the pdf file without leaving the `mes-aides` world ?